recipes/cppserver/all/conanfile.py

```diff
@@ -0,0 +1,73 @@
+from conans import ConanFile, CMake, tools
+import os
+import glob
+
+
+class CppServer(ConanFile):
+    name = "cppserver"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/chronoxor/CppServer"
+    description = "Ultra fast and low latency asynchronous socket server and" \
+        " client C++ library with support TCP, SSL, UDP, HTTP, HTTPS, WebSocket" \
+        " protocols and 10K connections problem solution."
+    topics = ("network", "socket", "async", "low-latency")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"fPIC": [True, False],
+               "shared": [True, False]}
+    default_options = {"fPIC": True,
+                       "shared": False}
+    requires = ["asio/1.17.0", "openssl/1.1.1g", "cppcommon/cci.20201104"]
+    generators = "cmake", "cmake_find_package"
+    exports_sources = ["patches/**", "CMakeLists.txt"]
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def _configure_cmake(self):
+        if not self._cmake:
+            self._cmake = CMake(self)
+            self._cmake.definitions["CPPSERVER_MODULE"] = "OFF"
+            self._cmake.configure(build_folder=self._build_subfolder)
+        return self._cmake
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = glob.glob("CppServer-*")[0]
+        import time
+        time.sleep(30)
```

Comment:
Why was this necessary?
```suggestion
```

Reply:
Oh, boy, sorry for that. It slipped in.

I have an ongoing issue with an antivirus here on my notebook which requires such kind of dirty tricks to bypass.

Reply:
Don't worry. At one time, we have all met a hard to understand set-up.

Maybe you can create a little `post_source` hook with a sleep on your problematic setup?
https://docs.conan.io/en/latest/extending/hooks.html